src/assets/wise5/authoringTool/peer-grouping/PeerGroupingLogic.ts

```diff
@@ -3,7 +3,12 @@ export interface PeerGroupingLogic {
   value: string;
 }
 
+export const DIFFERENT_IDEAS_NAME = $localize`Different Ideas`;
+export const DIFFERENT_IDEAS_REGEX = /differentIdeas\("(\w+)",\s*"(\w+)"\)/g;
+export const DIFFERENT_IDEAS_VALUE = 'differentIdeas';
+
 export const availableLogic: PeerGroupingLogic[] = [
```

Comment:
Change to upper-case for consistency?
```suggestion
export const AVAILABLE_LOGIC: PeerGroupingLogic[] = [
```